docs/querying/sql-functions.md

```diff
@@ -885,7 +885,7 @@ Calculates the base-10 of the numeric expression.
 
 ## LOOKUP
 
-`LOOKUP(<CHARACTER>, <CHARACTER>)`
+`LOOKUP(<CHARACTER>, <CHARACTER>, [<CHARACTER>])`
```

Comment:
nit: 
```suggestion
`LOOKUP(<CHARACTER>, <CHARACTER>[, <CHARACTER>])`
```

although looking at this page, this isn't actually consistent across all functions... i think this form makes more sense because the comma wouldn't be there unless the argument is there, but :shrug: